boards/st/stm32h573i_dk/stm32h573i_dk.dts

```diff
@@ -81,7 +81,7 @@
 	div-m = <5>;
 	mul-n = <96>;
 	div-p = <2>;
-	div-q = <2>;
+	div-q = <6>;
```

Comment:
Did you check that changing the pll1_q output freq does not disturb other peripherals that have pll1_q as clock source

Reply:
Yes, i checked it. The pll1_q  output frep is only used by the peripheral FDCAN  on stm32h573i_dk.